jobs/JWAVE_INIT

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 date
+export RUN_ENVIR=${RUN_ENVIR:-"nco"}
```

Comment:
Need to insert the following lines on this script some where before ROTDIR is first referred to.

if [ $RUN_ENVIR = "nco" ]; then
    export ROTDIR=${COMROOT:?}/$NET/$envir
fi

Reply:
Please see   JGLOBAL_FORECAST   as an example

Reply:
Completed